frontend/packages/console-app/src/components/quick-starts/QuickStartController.tsx

```diff
@@ -28,6 +34,22 @@ const QuickStartController: React.FC<QuickStartControllerProps> = ({ quickStart
     (task, index) => activeQuickStartState[`taskStatus${index}`],
   ) as QuickStartTaskStatus[];
 
+  const [contentRef, setContentRef] = React.useState<HTMLDivElement>(null);
+  const ref = React.createRef<HTMLDivElement>();
+  const shadows = useScrollShadows(contentRef, 100);
+
+  React.useEffect(() => {
+    ref.current !== contentRef && setContentRef(ref.current);
+  }, [ref, contentRef]);
+
+  React.useEffect(() => {
+    setShadows(shadows);
+  }, [shadows, setShadows]);
```

Comment:
You won't need these hooks if you move the `useScrollShadows` to `QuickStartPanelContent`. Furthermore the first `useEffect` is invalid because `ref` is a constant and would never update even if the `ref.current` value changed.

Reply:
updated